tests/testthat/test-ui-correlation.R

```diff
@@ -11,4 +11,6 @@ test_that("cXcorrelation2", {
 
 test_that("cXcorrelation3", {
     check_ui_test(cXcorrelation3())
+
+    warning("Clicking on any boxes after ordering labels will throw JS console errors")
```

Comment:
It isn't clear what you mean by boxes - I think you mean the chart.  Update the message to 'Clicking on the chart ...'